src/network/connection.cpp

```diff
@@ -1121,7 +1121,7 @@ bool UDPPeer::processReliableSendCommand(
 	u16 packets_available = toadd.size();
 	/* we didn't get a single sequence number no need to fill queue */
 	if (!have_initial_sequence_number) {
-		LOG(derr_con << m_connection->getDesc() << "Ran out of sequence numbers!" << std::endl);
+		verbosestream << m_connection->getDesc() << "Ran out of sequence numbers! (peer id: " << c_ptr->peer_id << ")" << std::endl;
```

Comment:
on second thought this is maybe less severe than it sounds because the window being full is quite a normal condition during bulk transfers...

Reply:
IMO, we should have everything needed to link up a networking log to a peer/event in our log, severe or not. This would help analyze issues specific to that peer/event and help in future debugs.

Reply:
I'm talking about the changed logging target.